pom.xml

```diff
@@ -1,53 +1,80 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-
-  <groupId>com.google.sps</groupId>
-  <artifactId>step240-2020</artifactId>
-  <version>1.0</version>
-  <packaging>war</packaging>
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.3.3.RELEASE</version>
+    <relativePath/> <!-- lookup parent from repository -->
+  </parent>
+  <groupId>com.google.graphgeckos</groupId>
+  <artifactId>dashboard</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <name>dashboard</name>
+  <description>Demo project for Spring Boot</description>
 
   <properties>
-    <failOnMissingWebXml>false</failOnMissingWebXml>
+    <java.version>1.8</java.version>
   </properties>
 
   <dependencies>
     <dependency>
-      <groupId>javax.servlet</groupId>
-      <artifactId>javax.servlet-api</artifactId>
-      <version>4.0.1</version>
-      <scope>provided</scope>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-data-rest</artifactId>
```

Comment:
Why do we need this? Could we maybe add this dependency later when the functionality that depends on it is added. Also for other dependencies.
c.f. `mvn spring-boot:run` runs just fine without this.

Reply:
The list of dependencies in pom.xml is a standard list for Spring Web App generated by Spring Boot Initializr. This one, for example, is [Spring Data REST](https://spring.io/projects/spring-data-rest). This dependency is needed to access its features, not to start the app.
I think, it would be way more convenient if we could start working, having this dependencies list, and delete unused  ones after we finish project or at least decide which of them we want to use.

Reply:
There are several of these dependencies that I think make sense to stay, like:
* spring-boot-starter-web
* spring-boot-devtools
* spring-boot-maven-plugin
* spring-boot-starter-test

For the others, it's a great idea to look into them, discuss their advantages and disadvantages and decide which one to use for what, but that doesn't need them to be listed here.

I believe it would be better to introduce them along with the code that starts using them.
Deleting unused dependencies after finishing the project or after deciding which to use is really the wrong way around this: you _first_ need to have a firm understanding of which exactly you need for a component of the system _before_ you start writing the code for it.
In particular for Spring Data REST, I'm curious how are you going to incorporate its functionality into the app? What is it for?

Reply:
Makes sense to me, I'll remove them